docs/general/performance.md

```diff
@@ -40,6 +40,8 @@ Decimating your data will achieve the best results. When there is a lot of data
 
 There are many approaches to data decimation and selection of an algorithm will depend on your data and the results you want to achieve. For instance, [min/max](https://digital.ni.com/public.nsf/allkb/F694FFEEA0ACF282862576020075F784) decimation will preserve peaks in your data but could require up to 4 points for each pixel. This type of decimation would work well for a very noisy signal where you need to see data peaks.
 
+Line charts are able to do [automatic data decimation during draw](#automatic-data-decimation-during-draw), when certain conditions are met. You should still consider data decimation for best results.
```

Comment:
We might be able to add a bit more detail in the second sentence. How about something like: "You should still consider decimating data yourself before passing it in for maximum performance since the automatic decimation occurs late in the chart life cycle"

Reply:
done